test/Interop/C/bounds-safety/Inputs/bounds-attributed-function.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
Can you leave a comment here summarizing the main combinations of language features you had though of testing here? Just so that it's a bit easier to get a sense of what you were thinking when you wrote these tests, and make better guesses as to what coverage might have been missed.